ngraph/core/reference/include/ngraph/runtime/reference/convert.hpp

```diff
@@ -19,8 +19,8 @@ namespace ngraph
             {
                 inline void set_u1(uint8_t* buf, size_t idx, uint8_t val)
                 {
-                    const int byte_idx = idx / 8;
-                    const int bit_idx = 7 - (idx % 8);
+                    const uint32_t byte_idx = idx / 8;
```

Comment:
Why not `size_t`?

Reply:
Yeah, I'm just not in a habit of using `size_t`. It's a better type for `byte_idx` - changed in this [commit](https://github.com/openvinotoolkit/openvino/pull/5914/commits/366ee072816507717bf395e88dccb78470c1b332)